packages/pluggableWidgets/image-web/src/components/Image/ui.tsx

```diff
@@ -62,7 +63,7 @@ function ContentGlyphicon(props: ImageContentGlyphicon): ReactElement {
 
     return (
         <span
-            className={classNames("glyphicon", props.icon)}
+            className={classNames(props.isGlyph === true ? "glyphicon" : "", props.icon)}
```

Comment:
can we rewrite this using object literal?
```suggestion
            className={classNames(props.icon, { glyphicon: props.isGlyph })}
```